script/api.py

```diff
@@ -0,0 +1,62 @@
+import os
+
+import psycopg2
+from flask import Flask, jsonify, request, session
+
+app = Flask(__name__)
+
+hostname = os.getenv('POSTGRES_HOST')
+database = os.getenv('POSTGRES_DB')
+username = os.getenv('POSTGRES_USER')
+
+port_id = 5432
+conn = psycopg2.connect(
+    host=hostname,
+    dbname=database,
+    user=username,
+    port=port_id
+)
+
+cur = conn.cursor()
+
+
+@app.route('/login', methods=['POST'])
+def login():
+
+    email = request.json.get('email')
+    passwrd = request.json.get('password')
+    cur.execute("SELECT * FROM users WHERE email='{}' ".format(email))
+    user_email = cur.fetchall()
+    if len(user_email) > 0:
+        cur.execute(
+            "SELECT * FROM users WHERE email='{}' AND password=crypt('{}',password)".format(email, passwrd))
+        user = cur.fetchall()
+
+        if len(user) > 0:
+            print(user)
+            id = user[0][0]
+            if 'user_id' in session:
+                return jsonify({"username": user[0][1]})
+            else:
+
+                session['user_id'] = id
+                return jsonify({"username": user[0][1]})
+        else:
+            return jsonify({"status": 401, "message": "The password that you've entered is incorrect. "})
```

Comment:
Instead of returning json directly with status inside the json body, we need to send the status code in headers.
Find about sending HTTPResponses in Flask.